README.md

```diff
@@ -1 +1,48 @@
-# Conference-Page
\ No newline at end of file
+# Conference-Page
+
+> First Capstone Project From Microverse
+
+## Built With
+
+- HTML5
+- CSS3
+- JavasScript
+- Bootstrap 5
+
+## Live Demo (if available)
+
+- [Demo video](https://drive.google.com/file/d/1RA2RLYQkh4YM6ZDvtryAswTmXHHzqT2O/view?usp=sharing)
+
+## Prerequisites
+
+- Needs Bootstrap 5 installed in the folder in order to display it correctly.
+
+### Deployment
+
+https://swarzstein.github.io/Conference-Page/
```

Comment:
- [OPTIONAL] I appreciate that you implemented the changes 💯 I noticed that the deploy link is broken, kindly fix it.
> But for the hard work you've done, your project deserves to be approved 🎉 I am sure that you will fix this minor issue